templates/definition/meter/enphase.yaml

```diff
@@ -27,9 +27,37 @@ render: |
     auth:
       type: bearer
       password: {{ .token }}
-    insecure: true
+    insecure: true 
     {{- end }}
     jq: .consumption[] | select(.measurementType == "net-consumption").wNow
+  currents:
+    - source: http
+      uri: http://{{ .host }}/production.json?details=1
+      {{- if .token }}
+      auth:
+        type: bearer
+        password: {{ .token }}
+      insecure: true
+      {{- end }}
+      jq: if .consumption[] | select(.measurementType == "net-consumption").activeCount >= 1 then .consumption[] | select(.measurementType == "net-consumption").lines[0].rmsCurrent else 0 end
```

Comment:
@pdeliot something like this:

jq: if (( .consumption[] | select(.measurementType == "net-consumption").activeCount >= 1 ) and ( .consumption[] | select(.measurementType == "net-consumption").lines | length >= 1 )) then .production[] | select(.measurementType == "net-consumption").lines[0].rmsCurrent else 0 end